src/test/java/uk/gov/hmcts/reform/sscs/service/NotificationServiceTest.java

```diff
@@ -2605,6 +2605,16 @@ protected static void verifyExpectedErrorLogMessage(Appender<ILoggingEvent> mock
         assertTrue(logEvents.stream().filter(logEvent -> logEvent.getFormattedMessage().contains(ccdCaseId)).count() >= 1);
     }
 
+    protected static void verifyExpectedInfoLogMessage(Appender<ILoggingEvent> mockAppender, ArgumentCaptor captorLoggingEvent, String ccdCaseId, String errorMessage) {
+        verify(mockAppender, atLeastOnce()).doAppend(
```

Comment:
This looks like repeated code from the method above with a different log level.
Since there are already 4 parameters to the method, it would take a few extract methods to resolve this. 
Ignore the comment if it proves too difficult.

Reply:
Added an extra param